src/controllers/controllermanager.cpp

```diff
@@ -392,7 +392,17 @@ void ControllerManager::slotSavePresets(bool onlyActive) {
         if (onlyActive && !pController->isOpen()) {
             continue;
         }
+        ControllerPresetPointer pPreset = pController->getPreset();
+        DEBUG_ASSERT(!pPreset);
+
         QString deviceName = sanitizeString(pController->getName());
+        if (!pPreset->isDirty()) {
```

Comment:
Should this be a `VERIFY_OR_DEBUG_ASSERT`?

Reply:
No, this is always called even if the mapping was not changed right now.

Reply:
okay